section2/reflection.md

```diff
@@ -1,13 +1,47 @@
 ## Section 2 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+  I'm definitely inclined to wait to ask questions. I really enjoy the process of figuring out something myself, which can be a strength in moderation, but problematic if I devote too much time to figuring out a solution. Going through Turing I'm going to make an effort to only spend that 20-30 minutes on a problem before reaching out for help (if I don't make progress).
```

Comment:
Good plan! Generally, I try to time-box the time spent "spinning my wheels" to about an hour max before looking for an alternative strategy.